### PR TITLE
Make Wanderers hostile when attacking Pug ships

### PR DESCRIPTION
**Pull request by 10010101001**
The Arfectas are too easy to cap at the moment. Also, after Rek specifically tells you not to offend the Pug, it seems strange that the Wanderers will stand by while you attack the Arfectas.

### CODE REVIEW

**Comment by Darcman99** [comment]
~~But then how will i get my OP ships and do the campaign too?~~

**Comment by tehhowch** [comment]
Seems this is just a portion of #3583 ? Specifically, 888b6526bccbef8cb06be46501fe46a26ed252d4 ? I'm more inclined to push that specific commit than a new PR that made no reference to the existing PR or discussion about this topic.

**Comment by Pointedstick** [comment]
I'm in favor of shutting down the Arfecta farming, FWIW. It's especially weird that the Wanderers permit it after specifically asking you not to.

File: data/wanderers start.txt [hunk 1]
```diff
@@ -2439,7 +2439,8 @@ mission "Wanderers Ap'arak 3: Pug"
 		government "Pug"
 		ship "Pug Arfecta" "Hork Ekekek Ptui"
 		ship "Pug Arfecta" "Graak Sput Kchoo"
-
+	on complete
+		"reputation: Wanderer" <?= -1
```
[the remark this answers is not in the document]
Comment (10010101001) [reply data/wanderers start.txt]: Done. Also added a dialog

File: data/wanderers start.txt [hunk 2]
```diff
@@ -2439,7 +2439,13 @@ mission "Wanderers Ap'arak 3: Pug"
 		government "Pug"
 		ship "Pug Arfecta" "Hork Ekekek Ptui"
 		ship "Pug Arfecta" "Graak Sput Kchoo"
+		
+	on complete
+		"reputation: Wanderer" <?= -1
+		dialog
+			`After destroying the Pug ships, a Wanderer ship hails you. " We have warned you against [attacking, angering] the Pug. This behavior is [unacceptable, untolerable]."`
```
Comment (Pointedstick) [inline data/wanderers start.txt]: Maybe change "untolerable" (which isn't a word) to "unforgivable", which might make more sense given that they then immediately become hostile.

Comment (Zireael07) [reply data/wanderers start.txt]: They probably meant 'intolerable' https://www.merriam-webster.com/dictionary/intolerable but unforgivable is a good synonym too

Comment (10010101001) [reply data/wanderers start.txt]: Fixed

**Pull request closed without merging**